src/StoreApi/Routes/V1/Checkout.php

```diff
@@ -92,11 +92,11 @@ public function get_args() {
 							],
 						],
 					],
-					$this->schema->get_endpoint_args_for_item_schema( \WP_REST_Server::CREATABLE )
+					$this->schema->get_endpoint_args_for_item_schema()
 				),
 			],
 			'schema'      => [ $this->schema, 'get_public_item_schema' ],
-			'allow_batch' => [ 'v1' => true ],
```

Comment:
Is this a breaking change? Can we implement rate limiting for batch requests?

Reply:
We don't use batch for Checkout, but clearly, I wasn't thinking straight disabling this, as others might. Reverting it.

I discussed this with Nadir a while back, releasing rate limiting in steps, first for the main request, gathering some feedback, and then with batch support. It is a scary feature when generalized to work in the wild, and it can break the most sensitive action: actual purchase.